src/main/java/org/sagebionetworks/web/client/security/AuthenticationControllerImpl.java

```diff
@@ -388,7 +389,6 @@ public void onSuccess(UserProfile result) {
               ginInjector.getGlobalApplicationState().refreshPage();
             }
             checkForQuarantinedEmail();
-            clearQueryClientCache();
```

Comment:
This callback is within a call to `initializeFromExistingAccessTokenCookie`, which unconditionally clears the cache, so this call is redundant.